### PR TITLE
Update session dg

### PR DESCRIPTION
**Pull request by ldinghan**

### CODE REVIEW

File: docs/diagrams/SessionClassUML.puml [hunk 1]
```diff
@@ -39,8 +39,8 @@ class Person {
 }
 
 Session *-- "1" SessionNumber: sessionNumber
-Session *-- "1" SessionStudents: sessionStudents
+Session *-- "1" StudentSet: sessionStudents
```
Comment (raydenlim) [inline docs/diagrams/SessionClassUML.puml:42]: rmb to update the image also :)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -360,6 +360,38 @@ This process is summarised in the activity diagram below
...
+Step 4:
+Finally, the `TakeAttendanceCommand` triggers the `Model` to update the filtered session list using the `updateFilteredSessionList` to display all sessions.
+
+
+#### Design Considerations:
+**Aspect: How models are modified to store changes**
+
+* **Alternative 1 (current choice):** Cause updates to both `Session` and `Person` when updating the attendance status of a student to a specified session.
+    * Pros: Allows for ease of future enhancements since we can cause the same update from either `Session` or `Person` involved.
+
```
Comment (raydenlim) [inline docs/DeveloperGuide.md]: No cons? haha
E.g more storage space used.

**Review by raydenlim — COMMENTED** [review]
LGTM, apart from some minor issues

File: docs/diagrams/SessionClassUML.puml (hunk 1 above)
Comment (ldinghan) [reply docs/diagrams/SessionClassUML.puml:42]: Yup, missed that out! Thanks

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -360,6 +360,38 @@ This process is summarised in the activity diagram below
 
 ![Delete Activity Diagram](images/DeleteStudentActivityDiagram.png)
 
+#### Take Attendance Feature
+This section explains the implementation of the Take Attendance feature via the `takeattendance` command.
+The `TakeAttendanceCommand` causes the specified `Person` to be added to the specified session.
```
Comment (Respirayson) [inline docs/DeveloperGuide.md:398]: Might be good to also mention the parameters that the command will take in

**Review by Respirayson — APPROVED** [review]
LGTM. one small nit tho

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (ldinghan) [reply docs/DeveloperGuide.md:398]: Sounds good!

**Pull request merged**